napari/layers/labels/labels.py

```diff
@@ -612,12 +612,7 @@ def _is_default_colors(self, color):
 
         if not np.allclose(color[None], [0, 0, 0, 1]):
             return False
-        if not np.allclose(
-            color[self.colormap.background_value], [0, 0, 0, 0]
-        ):
-            return False
-
-        return True
+        return np.allclose(color[self.colormap.background_value], [0, 0, 0, 0])
```

Comment:
Maybe combine with `if` above?

```python
return (
    np.allclose(color[None], [0, 0, 0, 1])
    and np.allclose(color[self.colormap.background_value], [0, 0, 0, 0])
)
```

Reply:
There is also one additional if above. Should I also merge it?
```
return (
	{None, self.colormap.background_value} == set(color.keys())
    and np.allclose(color[None], [0, 0, 0, 1])
    and np.allclose(color[self.colormap.background_value], [0, 0, 0, 0])
)
```